src/routes/init/helpers.ts

```diff
@@ -96,13 +95,12 @@ export async function getGithubUser() {
 
 export type User = {
     github: GithubUser | null;
-    appwrite: AppwriteUser | null;
 };
 
 export async function getUser(): Promise<User> {
-    const [github, appwrite] = await Promise.all([getGithubUser(), getAppwriteUser()]);
+    const github = await getGithubUser();
 
-    return { github, appwrite };
```

Comment:
Aren't there things that depend on `appwrite`? Like:

https://github.com/appwrite/website/blob/637a297fe2cdb68b92041600f574a1983ad1a5e2/src/routes/init/tickets/get-ticket-doc/%2Bserver.ts#L29-L35

and

https://github.com/appwrite/website/blob/637a297fe2cdb68b92041600f574a1983ad1a5e2/src/routes/init/tickets/get-ticket-doc/%2Bserver.ts#L118-L131